Cargo.toml

```diff
@@ -81,6 +81,11 @@ version = "2"
 optional = true
 version = "1"
 
+# Public: Used in trait impls on `Uuid`
+[dependencies.defmt]
```

Comment:
Since the `defmt` library is unstable, we'll need to jump through a few extra hoops to support it directly. If you follow the example of `zerocopy` we'll need to add a l[ine like this to our `lib.rs`](https://github.com/uuid-rs/uuid/blob/main/src/lib.rs#L215-L216) to ensure users also opt-in to an unstable feature flag to use `defmt`:

```rust
#[cfg(all(not(uuid_unstable), feature = "defmt"))]
compile_error!("The `defmt` feature is unstable and may break between releases. Please also pass `RUSTFLAGS=\"--cfg uuid_unstable\"` to allow it.");
```